include/common_components.php

```diff
@@ -1,24 +1,384 @@
     ";
+  }
+
+// --------------------------------------------------------------------------
+
+function echoHorizontalBar($heading, $subheading){
+ echo " 
+    <section>     
+            <nav>
+                <header>$heading
+                    <p> $subheading</p>
+                </header>
+                <div class='flex-item-horizontalMenu'>
+                    <ul>                         
+                        <a href='#'> <img class = 'notificationIcon' src= '/images/notification.png' alt='notification'></a> 
+                        <li><a href='#'>Students</a></li>
+                        <li><a href='#'>Calendar</a></li> 
+                        <li><a href='#'>Home</a></li>                        
+                    </ul> 
+                </div>                
+            </nav>      
+";
 }
 
+// --------------------------------------------------------------------------
+
 function echoFooter(){
-    echo"  <footer></footer>                            
+    echo"  
+                    </section>
+                </section>  
+                <footer></footer>                            
            </body>    
         </html>
        ";
+}
+
+// list_students.php file
+// --------------------------------------------------------------------------
+
+function echoSearchBar($heading){
+    echo"
+    <div class='flex-container-searchBarButtons'>
+        <div>
+            <h2>$heading</h2>
+            <div class='flex-item-SearchBar'>
+                <form action='' class='search-bar'>
+                <input type='text' placeholder='Search' name='searchBar'>
+                <button class='searchB' type='submit'><img class= 'searchButton'src='/images/search.jpg' alt='search'></button>
+                </form>
+            </div> 
+        </div>     
+";    
+}
+
+// --------------------------------------------------------------------------
+
+function addButtons($heading){
+    echo"
+        <div class='flex-item-buttons'>              
+            <button onclick= document.getElementById('id01').style.display='block' style='width:fit-content;'>$heading</button>
+        </div>
+    </div>    
+";    
+}
+
+// --------------------------------------------------------------------------
+
+function echoStudentTable($students, $credits, $classes, $days){
+    echo"
+    <div class='studentTable'>
+        <table id='tableContainer'>
+            <tr>
+                <th>Name</th>
+                <th>Email</th>
+                <th>Phone</th>
+                <th>ELO</th>
+                <th>Credits</th>
+                <th>Classes</th>
+                <th>Next Class in</th>                   
+            </tr>
+";        
+
+        $i = 0;
+        foreach($students as $student){                                                  
+        echo"
+        <tr>
+            <td>";passVariableThroughLink('student_profile.php', $student['Student_Id'], $student['First_Name']." ".$student['Last_Name']);echo"</td>
```

Comment:
Yes! But I deleted the function. I had it because I was planning to pass a class name as argument to give different CSS styling to the links but I don't think I'm doing that or will do it later